src/conn/conn_open.c

```diff
@@ -283,7 +283,7 @@ __wti_connection_workers(WT_SESSION_IMPL *session, const char *cfg[])
     WT_RET(__wti_prefetch_create(session, cfg));
 
     /* Start the checkpoint cleanup thread. */
```

Comment:
same here

Reply:
Done

Reply:
@ershov, the comment does not seem to be updated.